package.json

```diff
@@ -137,45 +137,64 @@
     "clean"
   ],
   "dependencies": {
-    "@unibeautify/beautifier-eslint": "^0.4.1",
-    "@unibeautify/beautifier-js-beautify": "^0.3.2",
-    "@unibeautify/beautifier-prettier": "^0.7.5",
-    "@unibeautify/beautifier-prettydiff": "^0.5.3",
-    "bluebird": "^3.5.1",
-    "cosmiconfig": "^4.0.0",
-    "eslint": "^4.19.1",
-    "js-beautify": "^1.7.5",
-    "prettier": "^1.11.1",
+    "@unibeautify/beautifier-black": "^0.1.1",
+    "@unibeautify/beautifier-brittany": "0.0.1",
+    "@unibeautify/beautifier-clang-format": "^0.2.0",
+    "@unibeautify/beautifier-csscomb": "^0.1.0",
+    "@unibeautify/beautifier-eslint": "^0.6.0",
+    "@unibeautify/beautifier-file": "^0.1.0",
```

Comment:
@Glavin001 what does this beautifier do?  Seems to have language support without any specified.

Reply:
@stevenzeck The file beautifier is for generic options: 
- end of line
- end with newline

See https://github.com/Unibeautify/website/pull/162

It's useful to put as last beautifier in `beautifiers` array and then you can change the line endings and add/remove newline at end of file.